css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  top: 50px;
+  left: 300px;
```

Comment:
Instead of doing this, you can set `width: 100%` so that the `text-align` declaration above this can do its job.